scripts/README.md

```diff
@@ -0,0 +1,83 @@
+# Scripts for creating custom docker images
+
+Each folder contains `.sh` script for creating either docker image or memgraph debian. Before running scripts, you need to change permissions on `build_` scripts with command:
+
+```
+sudo chmod 777 <SCRIPT_NAME>
+```
+
+Additional note, always check before running any scripts if your local user has the permission to read and write over the `resources` directory, which gets created in the `scripts` directory.
+
+## Creating Memgraph debian 11 package
+
+To create a Memgraph `.deb` file do the following steps:
+
+1. Run the `build_deb.sh` script as follows to create the deb from the GitHub repository:
+
+    ```console
+    $ sudo ./build_deb.sh {branch_name} {deb_name}
+    ```
+   - `{branch_name}`: any branch from the `memgraph` repo, e.g. `master` or `T1220-MG-properties-c++-api-bug`; for the script to pull and build `memgraph` from it
+   - `{deb_name}`: used to name the final  `.deb` package file: `memgraph-{deb_name}_amd64.deb`
+
+
+The script stores all `.deb` files in the `output_debian` folder.
+
+## Creating MAGE Docker image
+
+> ### This script releases docker images **only** for `amd64/linux` platform arhitectures. 
+
+1. Store the memgraph Debian package file in the `memgraph_deb_files` folder with the exact name: `memgraph-{deb_name}_amd64.deb` and run the script with the following command:
```

Comment:
Either change the folder name to `resources` or update the script to use `memgraph_deb_files`